sap-s4hana-soap/2.1/modules/ROOT/pages/sap-s4hana-soap-connector-studio.adoc

```diff
@@ -0,0 +1,175 @@
+|Unlisted services |List of services to add to the list of discovered services.
+|Service discovery resolution timeout | Connection timeout for service discovery.
+|Service discovery timeout unit | Time unit to use in the service discovery resolution timeout configuration.
+|Default headers |Custom headers to include in each request sent.
+|Default query parameter |Custom query parameters to include in each request sent.
+|TLS configuration |TLS configuration.
+|Proxy configuration |Configuration for executing requests through a proxy.
+|===
++
+. Click *OK* to save the global connector configuration.
+
+[[basic_authentication]]
+=== Basic Authentication
+
+Enter the following information on the *General* tab of the *Global Element Properties* screen to configure Basic authentication:
+
+* *Service Base URL* +
+URL of the SAP S/4HANA instance you want to manage
+* *Username* +
+User name used to initialize the session
+* *Password* +
+Password used to authenticate the user
+
+[[oauth-client-credentials]]
+=== OAuth 2.0 Client Credentials Authentication
+
+Enter the following required information on the *General* tab of the *Global Element Properties* screen to configure OAuth 2.0 client credentials authentication:
+
+* *Service base URL* +
+Base URL of the service provider of the SAP S/4HANA system to access from the connector. All other endpoints are derived from this base URL.
+* *Client ID* +
+OAuth client ID that is registered with the service provider.
+* *Client secret* +
+OAuth client secret that corresponds to the client ID and is registered with the service provider.
+
+=== Configure the SAP S/4HANA Cloud SOAP Listener Config
+
+Use the Global Element Properties configuration dialog to configure a global element for SAP S/4HANA Cloud SOAP Connector:
+
+. In your project, click the *Global Elements* tab at the base of the canvas.
+. In the *Global Configuration Elements* screen, click *Create*:
++
+image::sap-hana-source-global-config-wizard.png[]
++
+. In the *Choose Global Type* wizard, expand *Connector Configuration*, select *SAP S/4HANA Cloud SOAP Listener Config*, and click *OK*.
+. Configure the following values in the *General* tab:
+* *HTTP Listener* +
+   Reference to a global HTTPS Listener configuration, for example: `HTTPS_Listener_config`. 
+   This configuration must be secured by using **HTTPS protocol** in order to receive outbound messages from the SAP S/4HANA system.
+* *WSDL Path* +
+   Specify the paths to the WSDL definitions to use for resolving metadata keys.
+* *Encoding* +
+   Character encoding used in the messaging.
+. Click *OK* to save the global connector configuration.
+
+It is possible to share the same path between multiple sources, however, having multiple sources sharing the same path and listening to the same message type is not supported. 
```

Comment:
It might be more clear to say this: 

It is possible to share the same path between multiple sources, but sources with a shared path cannot listen to the same message type.